README.md

```diff
@@ -1,3 +1,2 @@
 "# SDPBootCamp" 
-"# SDPBootCamp" 
-"# SDPBootCamp" 
+

```

Comment:
FYI, you can use `git commit --allow-empty` to send empty commits for such purposes :)